src/module/cp/peeper.py

```
@@ -141,38 +151,38 @@ def send_now_board_with_verdict(message: RobotMessage):
 
     single_arg = "" if single_col else " --separate_cols"
     cached_prefix = get_cached_prefix('Peeper-Board-Generator')
-    run = call_lib_method(message,
+    run = _call_lib_method(message,
                           f"--now {single_arg} --verdict {verdict} --output {cached_prefix}.png")
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)